konf/site.yaml

```diff
@@ -149,6 +150,10 @@ production:
     if ($host = 'ubuntuserver.org' ) {
       rewrite ^ https://ubuntu.com/server$request_uri? permanent;
     }
+    if ($host = 'core.docs.ubuntu.com' ) {
+      rewrite ^(\/en)?(\/.*)$ https://ubuntu.com/core/docs$2? permanent;
+      rewrite ^ https://ubuntu.com/core/docs$request_uri? permanent;
```

Comment:
We don't need both, right? The previous one should cover all of the paths

Reply:
yep, just tested it. Removed.